pkg/channel/distributed/controller/testing/data.go

```diff
@@ -162,6 +162,18 @@ var (
 type ServiceOption func(service *corev1.Service)
 type DeploymentOption func(service *appsv1.Deployment)
 
+// Set The Service's DeletionTimestamp To Current Time
```

Comment:
I'm not sure this matters to the tests, but 1e9 isn't the Current Time (it's sometime in 2001)

Reply:
Yeah, I had just copied that from down below - you're right though and I'll fix it - good catch!